app-ui/src/pages/traces/ViewOptions.tsx

```diff
@@ -116,54 +155,56 @@ export function useViewOptions(): {
         </div>
       </div>
       <h2>Trace Badges</h2>
-      <h3>Choose which annotations bedges to display.</h3>
+      <h3>Customize the badges shown on traces in the sidebar.</h3>
       <div>
         <div>
           <input
             type="checkbox"
             name="viewOption"
             checked={viewOptions.showUserBadges}
-            id="expanded"
+            id="user-badges"
             onChange={(e) =>
               setViewOptions({
-                showUserBadges: e.target.checked  // Use the actual checkbox state
+                showUserBadges: e.target.checked, // Use the actual checkbox state
               })
             }
           />
           <AnnotationCounterBadge count={1} type="user" />
-          <label>User annotations.</label>
+          <label htmlFor="user-badges">User annotations.</label>
         </div>
 
         <div>
           <input
             type="checkbox"
             name="viewOption"
             checked={viewOptions.showAnalyzerModelBedge}
-            id="autocollapse-all"
+            id="analyzer-model-badges"
             onChange={(e) =>
               setViewOptions({
-                showAnalyzerModelBedge: e.target.checked  // Use the actual checkbox state
+                showAnalyzerModelBedge: e.target.checked, // Use the actual checkbox state
               })
             }
           />
           <AnnotationCounterBadge count={1} type="analyzer-model" />
-          <label>Analyzer Model annotations.</label>
+          <label htmlFor="analyzer-model-badges">
+            Analyzer Model annotations.
+          </label>
         </div>
 
         <div className="badge-checkbox-group">
           <input
             type="checkbox"
             name="viewOption"
             checked={viewOptions.showGuardrailsErrorBadge}
-            id="autocollapse-test-traces"
+            id="guardrail-badges"
             onChange={(e) =>
               setViewOptions({
-                showGuardrailsErrorBadge: e.target.checked  // Use the actual checkbox state
+                showGuardrailsErrorBadge: e.target.checked, // Use the actual checkbox state
               })
             }
           />
           <AnnotationCounterBadge count={1} type="guardrails-error" />
-          <label>Guardrails annotations.</label>
+          <label htmlFor="guardrail-badges">Guardrails annotations.</label>
```

Comment:
Adding htmlFor correctly, allows you to also click the label for toggling the checkbox, not just the checkbox itself.